docs/about/team/internships.md

```diff
@@ -4,6 +4,11 @@ sidebar_position: 4
 
 # Internships at the AI Lab
 
+### [Werbrouck, Francois] (https://www.linkedin.com/in/francois-werbrouck-b17296202) - Base de données Hivers 2024
```

Comment:
The name don't fit properly try:
```
### [Werbrouck, Francois](https://www.linkedin.com/in/francois-werbrouck-b17296202) - Base de données Hivers 2024
```

Reply:
I updated it with a working link without the hexadecimal part (I updated my linkedin profile link after doing the pr....).